src/words.test.ts

```diff
@@ -0,0 +1,45 @@
+import { words } from './words';
+
+describe('words', () => {
+  it('should not send a message if the author is a bot', async () => {
+    const mockArgs = {
+      isBot: true,
+      content: '',
+      send: jest.fn(),
```

Comment:
```suggestion
      isBot: true,
      content: 'this will trigger guys',
      send: jest.fn(),
```

In this test case, we want to assert that `send` is not called when the author is a bot. This means that there should be *no other* variables that would prevent `send` from being called. Therefore, we should use a `content` value that *would* trigger a response. Otherwise, if our `bot` logic was off we would never know because the test also uses a null content value.